<launch>
  <!-- Launch the grid map tutorial demo node -->
  <node pkg="grid_map_yh" type="laserToMap" name="laserToMap" output="screen" >
  
	<param name="size0" value="10"/>
    <param name="size1" value="10"/>
    <param name="resolution" value ="0.2"/>
	<param name="robotHeight" value="1.0"/>
	<param name="velodyneHeight" value="1.1"/>

	<param name="fillRadius" value="0.8"/>
	<param name="rangeRadius" value="2.5"/>
	<param name="groundTolarance" value="0.05"/>

	<param name="readNum" value="1800"/>
	<param name="startIndex" value="0"/>
	
	<param name="loadVelodyneDirName" value="/media/yh/YH/DataSet/YQ21/2017_03_09/2017_03_09_drive_0004_sync/velodyne_points/PC090004VTK/"/>	
	<param name="loadPoseName" value="/media/yh/YH/DataSet/YQ21/2017_03_09/2017_03_09_drive_0004_sync/icp/icp.txt"/>


  </node>
  <!-- Launch the grid map visualizer -->
  <!--node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
    <rosparam command="load" file="$(find grid_map_yh)/cfg/pointsToMap_Viz.yaml" />
  </node-->


  <!-- Launch RViz with the demo configuration -->
  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find grid_map_yh)/cfg/pointsToMap.rviz" /-->

</launch>
